lib/src/player.dart

```diff
@@ -273,12 +273,11 @@ class Player with PlayerEventHandler implements PlayerApi {
       _invokeMethod<void>(Methods.setSubtitle, id);
 
   @override
-  Future<SubtitleTrack> get subtitle async =>
-      await _invokeObjectMethod<SubtitleTrack>(
+  Future<SubtitleTrack?> get subtitle async =>
+      _invokeObjectMethod<SubtitleTrack>(
```

Comment:
why was the `await` removed?

Reply:
Because it has no effect :/
Why do you want to keep it?